src/controls/QskSlider.cpp

```diff
@@ -32,7 +32,7 @@ class QskSlider::PrivateData
     QPointF pressedPos;
     qreal pressedValue;
     bool tracking : 1;
-    Qt::Orientation orientation : 2;
+    Qt::Orientation orientation : 3;
```

Comment:
Why do we need 3 bits ?

Reply:
First, Qt::Horizontal and Qt::Vertical are defined as 1 and 2, not 0 and 1.
Then enums in Visual Studio C++ apparently are signed (as opposed to with gcc and clang), which means that Qt::Vertical (10) is interpreted as -2 and not 2.

So the following code on Windows:
```
QSlider s(Qt::Vertical);
qDebug() << s.orientation();
```
... produces this output:
```
Qt::Orientation(-2)
```

In light of this I am not sure using bit fields for enumerations is worth it at all...

Reply:
Ah of course - it has to be "uint orientation : 2;"

Reply:
I agree, that this optimization is not that important - when not having more than one bit field it is even of no value. However it only requires one cast in the getter and isn't hard to achieve.

I noticed, that we have the same problem in QskPageIndicator.

Reply:
I fixed the bitfield issues in QskPageIndicator and QskSlider.
The reason for the crash - and how to avoid it best - remains open.